solutions/platinum/kattis-segmentintersection.mdx

```diff
@@ -0,0 +1,111 @@
+---
+id: kattis-segmentintersection
+source: Kattis
+title: Line Segment Intersection
+author: Rameez Parwez
+---
+
+[Unoffical Analysis](https://cp-algorithms.com/geometry/segments-intersection.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(T)$, where $T$ is the number of test cases.
```

Comment:
**Time Complexity:** $\mathcal{O}(1)$ for each test case.

Reply:
ok